datafusion/sqllogictest/test_files/create_external_table.slt

```diff
@@ -100,9 +100,17 @@ CREATE EXTERNAL TABLE t(c1 int) STORED AS CSV WITH LOCATION 'foo.csv';
 statement error DataFusion error: SQL error: ParserError\("Unexpected token FOOBAR"\)
 CREATE EXTERNAL TABLE t(c1 int) STORED AS CSV FOOBAR BARBAR BARFOO LOCATION 'foo.csv';
 
+# Missing partition column
```

Comment:
It would be good to add a positive test case of the new syntax to sqllogictests as well. In insert_to_external.slt, we can even fully end-to-end test creating the external table, inserting some data, and querying it back to ensure everything is working as expected.

Reply:
Positive case is already covered in the mod tests, don't think adding another one in create_external_table.slt would be necessary. However, I think e2e in insert_to_external.slt is a great idea.